net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="79.10.2"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gwenhywfar$secondaryArchSuffix = $portVersion
+	cmd:gct_tool = $portVersion
+	cmd:gsa = $portVersion
+	cmd:gwbuild = $portVersion
+	cmd:gwenhywfar_config = $portVersion
+	cmd:mklistdoc = $portVersion
+	cmd:typemaker = $portVersion
+	cmd:typemaker2 = $portVersion
+	cmd:xmlmerge = $portVersion
+	lib:libgwenhywfar$secondaryArchSuffix = $libVersionCompat
+	lib:libgwengui_cpp$secondaryArchSuffix = $libVersionCompat
+	lib:libgwengui_gtk3$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+   	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libgtk_3$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+    "
+
+PROVIDES_devel="
+	gwenhywfar${secondaryArchSuffix}_devel = $portVersion
+	devel:libgwenhywfar$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_cpp$secondaryArchSuffix = $libVersionCompat
+	devel:libgwengui_gtk3$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	gwenhywfar$secondaryArchSuffix == $portVersion base
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+    "
+
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gettext
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure --omit-dirs "binDir" ./configure --bindir=$prefix/bin --with-guis=gtk3 LDFLAGS="-lnetwork"
```

Comment:
I saw [here](https://github.com/aqbanking/gwenhywfar/blob/bfca8b8a500ffb9ff6cb54b8f6c35565113eb0ac/configure.ac#L855) that this apparently also supports Qt 5 in addition to (or instead of) GTK 3. I think Qt would be preferred on Haiku because it is integrated better.

Any reason why this uses GTK specifically?

Reply:
Ah, GnuCash uses GTK, right?

Reply:
Correct, GnuCash uses GTK3. But to be completely honest, I was getting an error when trying to build the Qt part of gwenhywfar and then I realized about GnuCash needing GTK3 anyway.

Reply:
It would be ideal to be able to split the packages (base, QT, GTK), or decide which part of it to build, like emerge on Gentoo does. But it is extra work and no one seems to need gwenhywfar on Haiku right now, except for Gnucash.